tracer/src/Datadog.Trace/Configuration/ConfigurationSources/NullConfigurationSource.cs

```diff
@@ -42,10 +42,14 @@ internal class NullConfigurationSource : IConfigurationSource, ITelemeteredConfi
     public ConfigurationResult<bool>? GetBool(string key, IConfigurationTelemetry telemetry, Func<bool, bool>? validator)
         => null;
 
-    public ConfigurationResult<IDictionary<string, string>>? GetDictionary(string key, IConfigurationTelemetry telemetry, Func<IDictionary<string, string>, bool>? validator)
+    public ConfigurationResult<IDictionary<string, string?>>? GetDictionary(string key, IConfigurationTelemetry telemetry, Func<IDictionary<string, string?>, bool>? validator)
         => null;
 
+<<<<<<< HEAD
```

Comment:
Conflict markers made it to Git server?

Reply:
Yes, made a mess of my sync with v3-main. Should be fixed now. Thanks for pointing it out.